src/Tokenizer.zig

```diff
@@ -535,8 +542,8 @@ pub const Token = struct {
     /// belong to the implementation namespace, so we always convert them
     /// to keywords.
     /// TODO: add `.keyword_asm` here as GNU extension once that is supported.
-    pub fn getTokenId(comp: *const Compilation, str: []const u8) Token.Id {
-        const kw = all_kws.get(str) orelse return .identifier;
+    pub fn getTokenId(comp: *const Compilation, str: []const u8, default: Token.Id) Token.Id {
```

Comment:
Noticed late but this will always return `.extended_identifier` if it is called on a unicode identifier. So instead of adding a default parameter this shouldn't be called at all.

Reply:
Addressed in https://github.com/Vexu/arocc/pull/129